### DISCUSSION TITLE
```platform.uno/docs```. RichTextBlock link is broken

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#13596] [General]
When I'am using the following link the menubar crashes.
https://platform.uno/docs/articles/implemented/windows-ui-xaml-controls-richtextblock.html

Chrome:
![image](https://github.com/unoplatform/uno/assets/69314237/186dbdac-6bdb-4b8e-8bda-71693a68b781)

Mozilla:
![image](https://github.com/unoplatform/uno/assets/69314237/b897b313-8139-4d3d-8520-6483561d05fb)

**Comment by morning4coffe-dev** [comment]
Thanks for the report! This page seems to be hidden in the menubar and the new search, but it is still indexed on search engines, which seems to cause this issue 🤔

**Comment by limeniye** [comment]
https://github.com/unoplatform/uno/issues/13729
